index.html

```diff
@@ -29,7 +29,7 @@ <h1>Welcome to Alain's CSD120 Website!</h1>
             <p>Hello, my name is Alain Arseneault, im 22 and enjoy gaming as a past time, typically this means i procrastinate frequently however I believe I've taken control of this</p>
             <p>This website will be the culmination of my learning over the next few months</p>
             <h2>My Favorite things</h2>
-            <a href="https://imgur.com/gallery/OJg6Jb6"><img src="images/puppy.jpg" alt="images/puppy.jpg"></a>
+            <a href="https://imgur.com/gallery/OJg6Jb6"><img src="images/puppy.jpg" width="200" alt="images/puppy.jpg"></a>
```

Comment:
I just added this so your puppy image wasn't so large on the page.  A better approach (now that you're working on the Embedding Multimedia module) would be to just resize the image to a more appropriate width